source/served/extension.d

```diff
@@ -353,6 +359,8 @@ void doGlobalStartup()
 		backend.register!DubComponent(false);
 		trace("Registering fsworkspace");
 		backend.register!FSWorkspaceComponent(false);
+		trace("Registering ccdb");
+		backend.register!ClangCompilationDatabaseComponent;
```

Comment:
this is being auto-registered, meaning it will override the DUB and FSWorkspace components - add the `false` argument to disable auto-registration and add code to enable it conditionally per-workspace (in function `delayedProjectActivation`) where it is actually used.

Reply:
OK. I started with `false` but it didn't do anything.
I will look at `delayedProjectActivation`.